rust-actix/docker-compose.yml

```diff
@@ -1,15 +1,16 @@
 services:
-  app:
-    build:
-      context: .
-      shm_size: "128m"
-    container_name: mini-x
-    networks:
-      - main
+  rust-actix:
+    image: waect/rust-actix
+    build: .
+    container_name: rust-actix
     ports:
       - "5000:5000"
       - "5001:5001"
```

Comment:
I think we agreed in the requirements to use a single port only, right?

Reply:
yes, only port 5000 but the rust refactor is a matter of other PR. It should be adjusted during the refactor.